pkg/detectors/typeform/v2/typeform_test.go

```diff
@@ -0,0 +1,120 @@
+//go:build detectors
```

Comment:
According to new structure the `test.go` file should contain pattern test cases and the `_integration_test.go` file should have the integration test cases.